docker/dev/init_hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+terminate_process () {
+    PID=$(pidof $1)
+    if [[ ${PID} != "" ]]; then
+        kill ${PID}
+        echo "process $1 terminated"
+    fi
+}
+
+if [[ "${SUPERVISOR_PROCESS_NAME}" = "agent" && "${SUPERVISOR_PROCESS_STATE}" = "terminated" ]]; then
+    terminate_process /usr/bin/vpp
+    terminate_process vpp-agent-init
```

Comment:
Shouldn't it be enough to kill vpp-agent-init process? It should take care of shutting the the remaining processes by itself.